source/configure/calls-deployment.rst

```diff
@@ -380,3 +386,32 @@ Can the traffic between Mattermost and ``rtcd``  be kept internal or should it b
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 When possible, it's recommended to keep communication between the Mattermost cluster and the dedicated ``rtcd`` service under the same private network as this can greatly simplify deployment and security. There's no requirement to expose ``rtcd``'s HTTP API to the public internet.
+
+Troubleshooting
+---------------
+
+Connectivity issues
+~~~~~~~~~~~~~~~~~~~
+
+If calls are failing to connect or timing out, it's likely the `RTC Server Port <plugins-configuration-settings.html#rtc-server-port-udp>`__ is not open or forwarded correctly. An easy way to check whether data can go through is to perform some tests using the ``netcat`` command line tool.
```

Comment:
Relative links need to include the parent directory in the path. For example: ``/configure/plugins-configuration-settings.html#rtc-server-port-udp`` to work.

Reply:
Interesting, I have been using this sort of links before (e.g. [here](https://github.com/mattermost/docs/blob/9a4734903daa9c5e3c61c44af32c95ad66d69895/source/configure/calls-deployment.rst?plain=1#L69)) and they are working fine as far as I can see. The reason I prefer this form vs the absolute path (`/configure/`) is that the latter breaks in the preview environment.

Reply:
If you've managed to get them working in RST files, I'm all for it (especially since the generated previews won't work).

Reply:
ah! These are same-page links, correct? As opposed to taking the user to a different page altogether?

Reply:
Actually most of them link to a different page but the resolution should work fine, as long as the relative path is correct. On the official docs it shouldn't matter which form we use unless we were to change directory names and such.